src/frameworks/wordpress/redis.md

```diff
@@ -52,21 +52,20 @@ Then commit the resulting changes to your `composer.json` and `composer.lock` fi
 To enable the WP-Redis cache the `object-cache.php` file needs to be copied from the downloaded package to the `wp-content` directory.  Add the following line to the bottom of your `build` hook:
 
 ```bash
-cp -r vendor/devgeniem/wp-redis-object-cache-dropin/object-cache.php wp/wp-content/object-cache.php
+cp -r wp-content/wp-redis-object-cache-dropin/object-cache.php web/wp/wp-content/object-cache.php
 ```
 
 It should now look something like:
 
 ```yaml
 hooks:
     build: |
-      mkdir -p wp/wp-content/themes
-      mkdir -p wp/wp-content/plugins
-      mkdir -p wp/wp-content/languages
-      cp -r plugins/* wp/wp-content/plugins/
-      cp -r themes/* wp/wp-content/themes/
-      cp -r languages/* wp/wp-content/languages/
-      cp -r vendor/devgeniem/wp-redis-object-cache-dropin/object-cache.php wp/wp-content/object-cache.php
+    set -e
+    cp -R web/wp/wp-content/plugins/* web/wp-content/plugins/
+    cp -R web/wp/wp-content/themes/* web/wp-content/themes/
+    cp -r wp-content/wp-redis-object-cache-dropin/object-cache.php web/wp/wp-content/object-cache.php
+    # Uncomment this line when you start adding extra language packs.
+    #cp -R web/wp/wp-content/languages/* web/wp-content/languages/
```

Comment:
Is it just a quirk of GitHub, or are the new lines not indented properly?  They should be 4 spaces in from `build`.

Otherwise this seems fine; thanks for the catch.

Reply:
Yes, obviously. Indenting should be 4 spaces of course.

Reply:
@Crell , fixed the indenting.